<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>921</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>921</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа №4</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Способ построения</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="AlgorithmTypeBox">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Каноническое уравнение</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Параметрическое уравнение</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Алгоритм Брезенхема</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Алгоритм средней точки</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Библиотечная функция</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>175</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>175</height>
         </size>
        </property>
        <property name="title">
         <string>Цвет</string>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>141</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Цвет рисования</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>100</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Цвет фона</string>
         </property>
        </widget>
        <widget class="QPushButton" name="changePenColorBtn">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>60</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Изменить</string>
         </property>
        </widget>
        <widget class="QPushButton" name="changeBackgroundColorBtn">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>130</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Изменить</string>
         </property>
        </widget>
        <widget class="QLabel" name="PenColor">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>60</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: black</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="backgroundColor">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>130</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: white</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>520</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Окружность</string>
         </attribute>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>300</width>
            <height>115</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>115</height>
           </size>
          </property>
          <property name="title">
           <string>Координаты центра</string>
          </property>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>121</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>X</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>40</y>
             <width>121</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Y</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QTextEdit" name="CircleCenterX">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>70</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QTextEdit" name="CircleCenterY">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>70</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="groupBox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>140</y>
            <width>300</width>
            <height>101</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>115</height>
           </size>
          </property>
          <property name="title">
           <string>Радиус</string>
          </property>
          <widget class="QTextEdit" name="CircleRadius">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>50</y>
             <width>271</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>250</y>
            <width>300</width>
            <height>120</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>120</height>
           </size>
          </property>
          <property name="title">
           <string>Параметры спектра</string>
          </property>
          <widget class="QTextEdit" name="CircleStep">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>70</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>121</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Шаг</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QTextEdit" name="CircleNumFigures">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>70</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_14">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>40</y>
             <width>121</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Кол-во фигур</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </widget>
         <widget class="QPushButton" name="plotCircle">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>390</y>
            <width>300</width>
            <height>31</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>Построить окружность</string>
          </property>
         </widget>
         <widget class="QPushButton" name="plotCircleSpectrum">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>430</y>
            <width>300</width>
            <height>31</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>Построить спектр</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Эллипс</string>
         </attribute>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>140</y>
            <width>300</width>
            <height>121</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>131</height>
           </size>
          </property>
          <property name="title">
           <string>Полуоси</string>
          </property>
          <widget class="QLabel" name="label_10">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>121</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Малая</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QTextEdit" name="SmallHalfAxis">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>70</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QTextEdit" name="BigHalfAxis">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>70</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_11">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>40</y>
             <width>121</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Большая</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>300</width>
            <height>115</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>115</height>
           </size>
          </property>
          <property name="title">
           <string>Координаты центра</string>
          </property>
          <widget class="QLabel" name="label_12">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>121</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>X</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="label_13">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>40</y>
             <width>121</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Y</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QTextEdit" name="EllipseCenterX">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>70</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QTextEdit" name="EllipseCenterY">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>70</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="groupBox_7">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>260</y>
            <width>300</width>
            <height>120</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>120</height>
           </size>
          </property>
          <property name="title">
           <string>Параметры спектра</string>
          </property>
          <widget class="QTextEdit" name="EllipseStep">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>70</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>121</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Шаг</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QTextEdit" name="EllipseNumFigures">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>70</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_15">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>40</y>
             <width>121</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Кол-во фигур</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </widget>
         <widget class="QPushButton" name="plotEllipse">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>400</y>
            <width>300</width>
            <height>31</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>Построить эллипс</string>
          </property>
         </widget>
         <widget class="QPushButton" name="plotEllipseSpectrum">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>440</y>
            <width>300</width>
            <height>31</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>Построить спектр</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearBtn">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string>Очистить плоскость</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background:white</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
